packages/app-desktop/InteropServiceHelper.ts

```diff
@@ -78,8 +78,12 @@ export default class InteropServiceHelper {
 					shim.setTimeout(async () => {
 						if (target === 'pdf') {
 							try {
+								const openDetails = '[].forEach.call( document.querySelectorAll( \'details\' ), el => el.setAttribute( \'open\', \'\' ) )';
```

Comment:
Why do we need this syntax `[].forEach.call`?

Reply:
I think this [answer](https://stackoverflow.com/questions/16053357/what-does-foreach-call-do-in-javascript) describes it better.
But other than that, there is no specific reason we should be doing that. Although I don't see any reason why we could just not do 
`document.querySelectorAll( 'details' ).forEach(el=>el.setAttribute( 'open')); I will fix this right now.